sklearn/mixture/tests/test_gmm.py

```diff
@@ -111,7 +121,11 @@ def test_lmvnpdf_full():
     reference = _naive_lmvnpdf_diag(X, mu, cv)
```

Comment:
Travis is not happy because _naive_lmvnpdf_diag creates diagonal covariances. I think you'll need more _naive_lmvnpdf_spherical (if it exists).